py/autoevals/oai.py

```diff
@@ -35,7 +35,8 @@ def open_cache():
 CACHE_LOCK = threading.Lock()
 
 
-def prepare_openai_complete(is_async=False, api_key=None):
+def prepare_openai_complete(is_async=False, api_key=None, base_url="https://api.openai.com/v1"):
```

Comment:
would rather not hardcode the API url here -- e.g. someone may monkey patch the library to override the default, and this would override that. Is there any issue with passing `None` into the `OpenAI` object by default?